src/server/mcp.ts

```diff
@@ -652,52 +703,92 @@ export class McpServer {
     description: string,
     paramsSchema: Args,
     annotations: ToolAnnotations,
-    cb: ToolCallback<Args>,
+    callback: ToolCallback<Args>,
```

Comment:
can we keep the same name please?

```suggestion
    cb: ToolCallback<Args>,
```